src/averageAges.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
+function hasChildren(people, woman) {
+  return people.some((person) => person.mother === woman.name);
+}
+
+function getMother(people, child) {
+  return people.find(mother => mother.name === child.mother);
+}
```

Comment:
better to make those arrow ones. There's a difference in the hoisting process. Take a look at [the article](https://www.educative.io/answers/function-declaration-vs-function-expression-in-javascript)

Reply:
not critical though